utils.js

```diff
@@ -137,63 +129,48 @@ const getUserTotalVotingPowerAtReferenceBlock = async (
   daoContract,
   tokenID,
   level,
-  userAddress,
-  isXTZ = false
```

Comment:
This should not be removed, i think some older code written previously is overwriting some latest changes.

Reply:
On it. Will fix this soon.